tests/framework/installer/ceph_installer.go

```diff
@@ -492,12 +502,15 @@ func NewCephInstaller(t func() *testing.T, clientset *kubernetes.Clientset, rook
 	if err != nil {
 		panic("failed to get kubectl client :" + err.Error())
 	}
+	// 0.8 is not able to handle the changed hostname until after this fix is backported.
+	// Then we can always change the hostname.
```

Comment:
Do we really need this comment here? As I understand we will immediately backport this fix after merge and make a release :wink:

Reply:
Yeah, should've been a comment in the PR instead of the code. I'll remove it.